modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,309 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-mtu.adoc
+
+[id="nw-cluster-mtu-change_{context}"]
+= Changing the cluster MTU
+
+As a cluster administrator, you can change the maximum transmission unit (MTU) for your cluster. The migration is disruptive and nodes in your cluster might be temporarily unavailable as the MTU update rolls out.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You identified the target MTU for your cluster. The correct MTU varies depending on the cluster network provider that your cluster uses:
+** *OVN-Kubernetes*: The cluster MTU must be set to `100` less than the lowest MTU value in your cluster.
+** *OpenShift SDN*: The cluster MTU must be set to `50` less than the lowest MTU value in your cluster.
+
+.Procedure
+
+To increase or decrease the MTU for the cluster network complete the following procedure.
+
+. To obtain the current MTU for the cluster network, enter the following command:
++
+[source,terminal]
+----
+$ oc describe network.config cluster
+----
+
+. Get the NetworkManager connection name:
+
+.. To find the primary network interface, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host ip route list match 0.0.0.0/0 | awk '{print $5 }'
```

Comment:
We are assuming ipv4 throughout the doc. I am not sure if we generally provide guidance for ipv4 only, but ipv6 would be slightly different on the commands (`ip -6 ... <ipv6 address>`). Just a heads up.

Reply:
@jcaamano, so that's interesting and I had not considered that; If you have an IPv4+IPv6 cluster, would you do the migration twice? I don't think we support IPv6-only clusters.

Reply:
Ah, I thought we did support IPv6-only clusters as we have CI for it but indeed I did not find it documented anywhere.